drawer.py

```diff
@@ -73,12 +75,25 @@ def make_matplotlib(self, dot_matrix, subplot, sequences):
             subplot - AxesSubplot object
             sequences - list of two objects Sequence being plotted
         """
+
         subplot.imshow(dot_matrix, cmap='Greys', interpolation='nearest')
         subplot.set_xlabel(sequences[1].name)
         subplot.set_ylabel(sequences[0].name)
-        if len(sequences[0].sequence) < 100:
+
+        if self.show_sequence == 'True':
             subplot.yaxis.set_major_locator(ticker.MultipleLocator(1))
             subplot.set_yticklabels('a' + sequences[0].sequence)
-        if len(sequences[1].sequence) < 100:
             subplot.xaxis.set_major_locator(ticker.MultipleLocator(1))
             subplot.set_xticklabels('a' + sequences[1].sequence)
+        elif self.show_sequence == 'False':
+            pass
+        elif self.show_sequence.isdigit():
```

Comment:
Yes, this is a good idea. Sorry I didn't check that before making a pull request.